### ISSUE TITLE
[train.py] cos_loss not included in running_loss — bug or intended?

### ISSUE THREAD

**Issue by wjh136** [hustvl/LightningDiT#72]
Hello, I have a small question while studying the train.py training script from your LightningDiT project. 
When using the VELOCITY prediction type with cos_loss enabled, I noticed that although the actual loss used for backpropagation is mse_loss + cos_loss, the running_loss used for logging seems to only accumulate mse_loss.item().
I was wondering if this is an intentional design choice or a possible oversight?

**Comment by JingfengYao** [comment]
Hi，

It can be said to be intentional. Because when I was optimizing LightningDiT's design, I fixed the feature space using SD-VAE. At this point, the MSE loss could directly reflect the model's performance (note that cross-tokenizer scenarios are not applicable). As for the cosine loss, being one of the tricks, it wasn't within the scope of my key focus at the time—I was still solely monitoring the trend of the MSE loss.
